core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -709,10 +728,8 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
     workflowsWithOutputs.map { case (workflowRecord, outputsResponse) =>
       val outputs = outputsResponse.outputs
       logger.debug(
-        s"attaching outputs for ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}: ${outputs.size} outputs"
-      )
-      logger.debug(
-        s"output expressions for ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}: ${outputExpressionMap.size} expressions"
+        s"attaching outputs for ${submissionId.toString}/${workflowRecord.externalId
+            .getOrElse("MISSING_WORKFLOW")}: ${outputExpressionMap.size} expressions, ${outputs.size} outputs"
```

Comment:
here, I combined two log lines into one to reduce overall logging volume. Will this have any repercussions?

Reply:
Can't speak to repercussions (are we watching that line specifically anywhere?) but this seems fine at first glance.

Is there any way to make it more obvious what an expression vs an output is? In your example they are all `1 expression , 1 output`

Reply:
I tweaked this to change "output" to "attribute values" in 4da55d5c8a620abe0dc7329105a96177bc64865c, hopefully that's a little more understandable